src/data/scripts/process_data.py

```diff
@@ -147,6 +148,9 @@ def processBuildingData() -> List[str]:
     # Add building grades:
     building_data = grade_buildings(building_data)
 
+    # Add building ward numbers under col: "Ward"
+    building_data = add_ward_number(building_data)
```

Comment:
If you added this to the data pipeline, you should re-run the data pipeline and committed the updated CSV files, as otherwise after this PR someone running the pipeline would see changes